### PR TITLE
Separate requirements install instruction

### PR DESCRIPTION
**Pull request by pdelboca**
A common issue for some installs are the requirements for Solr which are not mandatory if the user will run it using Docker.

This PR proposes a small change into the introduction of the instructions to install it from source to:

 - Explicitly name all the service required (I think it provide a good general view of the upcoming install process)
 - Separate each requirement in a different command so specifically link it to the service.
 - Cleans the old `who.ini` file section since it's no longer needed in CKAN 2.10.

### CODE REVIEW

File: doc/maintaining/installing/install-from-source.rst [hunk 1]
```diff
@@ -24,10 +24,26 @@ work on CKAN.
...
+The 3 main services (PostgreSQL, Solr and Redis) can be installed locally or dockerized. If
+you want to run them locally you will need to install the following dependencies.
+
+PostgreSQL requirements::
+
+    sudo apt-get install postgresql
+
+Solr requirements::
+
+    sudo apt-get install solr-tomcat openjdk-8-jdk
```
Comment (amercader) [inline doc/maintaining/installing/install-from-source.rst:42]: I think we should encourage people to use the Dockerized Solr version whenever possible, so let's remove these instructions for postgres and solr, as people will find them in their dedicated pages (actually check the postgres one). redis-server can be added back to the packages above

Comment (pdelboca) [reply doc/maintaining/installing/install-from-source.rst]: Done @amercader !

**Comment by pdelboca** [comment]
I have implemented your suggestions @amercader ! Let me know your thoughts!

File: doc/maintaining/installing/install-from-source.rst [hunk 2]
```diff
@@ -4,6 +4,8 @@
 Installing CKAN from source
 ===========================
 
+CKAN is a Python application that requires three main services: PostgreSQL, Solr and Redis.
+
 This section describes how to install CKAN from source. Although
 :doc:`install-from-package` is simpler, it requires Ubuntu 18.04 64-bit or
 Ubuntu 16.04 64-bit. Installing CKAN from source works with other
```
[the remark this answers is not in the document]
Comment (EricSoroos) [reply doc/maintaining/installing/install-from-source.rst:11]: There's also a place in the troubleshooting that mentions 16.04 and 18.04.

**Comment by EricSoroos** [comment]
We should probably merge this prior to 2.10 & backport

**Pull request merged**